### ISSUE TITLE
GUI Train error when using append

### ISSUE THREAD

**Issue by shnchh** [kahst/BirdNET-Analyzer#385]
**Describe the bug**
When using append model save mode, the trainer runs until completion, then produces the error screen along with a message "No such file or directory". No error message found in log other than a TensorFlow warning on deprecated cross entropy loss function. The GUI settings screen shows "no content" under error log file.
The tflite file is output but there is no labels and parameters file.

This issue doesn't happen if I choose replace and not append.

**To Reproduce**

1. Go to Train
2. Added two classes of .wav files.
3. Save as tflite file, append mode, everything else default, start training.

**Expected behavior**
No error 

**Desktop (please complete the following information):**
 - win 10 pro
 - Version 22h2, is 19045.4651

**Comment by Josef-Haupt** [comment]
Sadly I can't reproduce this bug on my end, please post a screenshot of all the options used in the training tab.

**Comment by shnchh** [comment]
![image](https://github.com/user-attachments/assets/b10f42c7-7083-4d4f-98d4-72168eede29f)
![image](https://github.com/user-attachments/assets/c7ca0e68-2d89-4c9c-a27a-49345e9d30ba)
![image](https://github.com/user-attachments/assets/e445893a-f376-4311-b093-0085d728a466)

**Comment by max-mauermann** [comment]
I also cant reproduce this error in the current version with the parameters seen in the screenshots.
Does it still occur on your system? If yes, would you mind sharing the data you use, this might also be something that causes the problem.

**Comment by shnchh** [comment]
I have currently organised my recordings into 3 folders: (I removed the bird species name as it is a sensitive species)
![image](https://github.com/user-attachments/assets/aa9f317f-1592-4c72-9e11-5f2301c0d0af)

Here is a sample recording from my "others" folder.
[Recording](https://drive.google.com/file/d/1qHsg9J8axYU9RorFMJc09uozkE6XVbLU/view?usp=sharing)

**Comment by larskristian02** [comment]
I am experiencing the exact same issue and can't seem to find a solution. Have you managed to fix it?

**Comment by shnchh** [comment]
I did not, but I managed to work around it by working directly with the command line interface and fixing some minor bugs in the source code. e.g. `python train.py --i "folder path" --o "outputfolder\\classifiername.tflite"
`

**Comment by larskristian02** [comment]
Thank you! I'm new to python, so i'm just wondering: do you still use the BirdNET gui application, or do you do the training manually in python?

**Comment by shnchh** [comment]
Nope you would not use the GUI. You can follow the instructions found on the GitHub birdnet homepage, under Step 6. setup (depending on your OS) and 7.1 CLI
